src/components/symptom-shark/presentational/BulletChart/BulletChart.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { DateRangeCoordinator, Layout, Section } from "../../../presentational";
+import BulletChart, { SymptomSharkBulletChartProps } from "./BulletChart";
+import { SymptomSharkVisualizationCoordinator } from "../../container";
+
+export default { title: "SymptomShark/Presentational/BulletChart", component: BulletChart, parameters: { layout: 'fullscreen' } };
+let render = (args: SymptomSharkBulletChartProps) => <Layout colorScheme="auto">
+    <DateRangeCoordinator intervalType={"Month"} variant="rounded">
+        <SymptomSharkVisualizationCoordinator previewState="default" showFilters>
+            <Section><BulletChart {...args} /></Section>
+        </SymptomSharkVisualizationCoordinator>
+    </DateRangeCoordinator>
+</Layout>
```

Comment:
Sorry to be a broken record, just want to mark the spots.  Missing semicolon here.  Also, the `args` param above can probably be dropped since it is never populated.